backend/database/migrations/versions/27f7d40373ea_add_student_simulation_instances.py

```
@@ -0,0 +1,45 @@
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('langchain_pg_collection',
+    sa.Column('name', sa.VARCHAR(), autoincrement=False, nullable=True),
+    sa.Column('cmetadata', postgresql.JSON(astext_type=sa.Text()), autoincrement=False, nullable=True),
+    sa.Column('uuid', sa.UUID(), autoincrement=False, nullable=False),
+    sa.PrimaryKeyConstraint('uuid', name='langchain_pg_collection_pkey'),
+    postgresql_ignore_search_path=False
+    )
+    op.create_table('langchain_pg_embedding',
+    sa.Column('collection_id', sa.UUID(), autoincrement=False, nullable=True),
+    sa.Column('embedding', pgvector.sqlalchemy.Vector(), autoincrement=False, nullable=True),
```

Ruff (0.13.1)

Undefined name `pgvector`

(F821)